test/check-cli

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/python3
 
+from unittest import skip
```

Comment:
Please use `import unittest` and the fully qualified name (`unittest.skip`) further down. It's hard for me to find the definition of things when the namespace is not attached to names.